go/mysql/ldapauthserver/auth_server_ldap.go

```diff
@@ -43,11 +43,12 @@ var (
 type AuthServerLdap struct {
 	Client
 	ServerConfig
-	Method         string
-	User           string
-	Password       string
-	GroupQuery     string
-	UserDnPattern  string
+	Method        string
+	User          string
+	Password      string
+	GroupQuery    string
+	UserDnPattern string
+	// staticcheck: var RefreshSeconds is of type time.Duration; don't use unit-specific suffix "Seconds" (ST1011)
```

Comment:
@sougou any thoughts on this? if this is intended to be number of seconds, then the way it is being used later is wrong.

Reply:
The naming and use match and indicate that these are seconds. This means the unit is wrong. We should change it to `int64`.